lib/service/storage/elasticsearch.ts

```diff
@@ -2461,10 +2656,20 @@ class ElasticSearch extends Service {
    * @returns {Promise.<{ items, errors }>
    */
   async mUpsert(
-    index,
-    collection,
-    documents,
-    { refresh, retryOnConflict = 0, timeout, userId = null } = {},
+    index: string,
+    collection: string,
+    documents: JSONObject[],
+    {
+      refresh,
+      retryOnConflict = 0,
+      timeout,
+      userId = null,
+    }: {
+      refresh?: boolean | "wait_for";
+      retryOnConflict?: number;
+      timeout?: string;
+      userId?: string;
+    } = {},
```

Comment:
```suggestion
    {
      refresh,
      retryOnConflict = 0,
      timeout,
      userId = null,
    }: Exclude<KRequestParams, "injectKuzzleMeta"> = {},
```
With new `KRequestParams` type we can compose type like it, to DRY